crates/telio-wg/src/wg.rs

```diff
@@ -718,8 +719,19 @@ impl State {
                 .unwrap_or_default();
         }
 
+        // Adapter doesn't keep track of mesh addresses, therefore they are not retrieved from UAPI requests
+        // so we need to keep track of it.
+        let mut new_interface = to.clone();
```

Comment:
instead of aditional clone, chage argument to this function to be a value type, and add mut to it. 
I see that in all uses of update, before sending to it we already do a clone. so this will reduce cloneing in general.

Reply:
changed